server/pkg/api/sessionHandlers.go

```diff
@@ -30,13 +30,15 @@ func sessionHandler(ctx *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	session.Expiration = time.Now().Add(time.Second * auth.SessionMaxAge)
+	session.RememberMe = r.URL.Query().Get("RememberMe")
```

Comment:
I don't think this needs to be here. We only want to set this flag on login and leave it as is afterwards.